internal/grpcutil/target.go

```diff
@@ -43,6 +43,9 @@ func split2(s, sep string) (string, string, bool) {
 // target}.
 func ParseTarget(target string) (ret resolver.Target) {
 	var ok bool
+	if strings.HasPrefix(target, "unix:") {
```

Comment:
Is this necessary for setting authority? Or something else?

Reply:
On further review this wasn't necessary. Removed this condition.